armory/scenarios/poison.py

```diff
@@ -205,14 +206,13 @@ def poison_dataset(self):
         self.record_poison_and_data_info()
 
     def record_poison_and_data_info(self):
-
         self.n_poisoned = int(len(self.poison_index))
         self.n_clean = (
-            len(self.y_clean) - self.n_poisoned
+            len(self.y_poison) - self.n_poisoned
         )  # self.y_clean is the whole pre-poison train set
```

Comment:
The comment is no longer relevant given you're using self.y_poison instead of self.y_clean

Reply:
Removed